src/Models/ProductModel.php

```diff
@@ -14,8 +13,58 @@ class ProductModel implements ProductModelInterface
      * ProductModel constructor.
      * @param $db
      */
-    public function __construct($db)
+    public function __construct(\PDO $db)
     {
         $this->db = $db;
     }
+
+    /**
+     * Gets all products from Database
+     * @return array|false depending on successful query or not
+     */
+    public function getAllProducts()
```

Comment:
if it returns as an array does this need an ": array" at the end?

Reply:
it returns an array if the query worked but false if it didn't